modules/serverless-config-high-workloads-serving.adoc

```diff
@@ -0,0 +1,71 @@
+* One activator pod can handle approximately 2500 requests per second before it starts to increase latency and, at some point, leads to errors.
+* One `3scale-kourier-gateway` or `istio-ingressgateway` pod can also handle approximately 2500 requests per second before it starts to increase latency and, at some point, leads to errors.
+* Each of the data-plane components consumes up to 1 vCPU of CPU for handling 2500 requests per second. Note that this highly depends on the payload size and the response times of the Knative Service backend.
+
+[IMPORTANT]
+====
+Fast startup and fast response-times of your Knative Service user workloads are critical for good performance of the overall system. The Knative Serving components are buffering incoming requests when the Knative Service user backend is scaling-up or when request concurrency has reached its capacity. If your Knative Service user workload introduces long startup or request-latency, it will either overload the `activator` component (when the CPU and memory configuration is too low) or lead to errors for the calling clients.
+====
+
+.Procedure
+
+* To fine-tune your installation, use the previous findings combined with your own test results to configure the `KnativeServing` custom resource:
++
+.A high workload configuration in KnativeServing CR
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  high-availability:
+    replicas: 2 <1>
+  workloads:
+    - name: component-name <2>
+      replicas: 2 <2>
+      resources:
+        - container: container-name
+          requests:
+            cpu: <3>
+            memory: <3>
+          limits:
+            cpu: <3>
+            memory: <3>
+  podDisruptionBudgets: <4>
+    - name: name-of-pod-disruption-budget
+      minAvailable: 1
+----
+<1> Set this parameter to at least `2` to make sure you always have at least two instances of every component running. You can also use `workloads` to override the replicas for certain components.
+<2> Use the `workloads` list to configure specific components. Use the `deployment` name of the component and set the `replicas` field. For components that use the HorizontalPodAutoscalers (`activator`, `webhook`, and `3scale-kourier-gateway`), the `replicas` field sets the minimum number of replicas. The actual number of replicas will depend on the CPU load and scaling done by the HPAs.  
+<3> Set the requested and limited CPU + memory according to at least the idle consumption while also taking the above findings and your own test results into consideration.
+<4> Adjust the `PodDistruptionBudgets` to a value lower than `replicas`, to avoid problems during node maintenance. The default `minAvailable` is set to `1`, so if you increase the desired replicas, make sure to also increase `minAvailable`.
```

Comment:
```suggestion
<4> Adjust the `PodDistruptionBudgets` to a value lower than `replicas`, to avoid problems during node maintenance. The default `minAvailable` is set to `1`, so if you increase the required replicas, make sure to also increase `minAvailable`.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#d